draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1397,70 +1419,58 @@ Internet-Draft                 ACTN POI                       July 2023
    abstraction is valid independently of the fact that optical
    transponders are physically integrated within the same WDM node or
    are physically located on a device external to the WDM node since it
-
-
-
-Peruzzini et al.       Expires January 7, 2024                [Page 26]
-
-Internet-Draft                 ACTN POI                       July 2023
-
-
    both cases the optical transponders and the WDM node are under the
    control of the same O-PNC.
 
-   The association between the Ethernet LTPs terminating the Ethernet
-   cross-layer links and the optical TTPs is reported using the Inter
-   Layer Lock (ILL) identifiers, defined in [RFC8795].
+   The association between the Ethernet or CBR client LTPs terminating
+   the Ethernet cross-layer links and the optical TTPs is reported using
+   the Inter Layer Lock (ILL) identifiers, defined in [RFC8795].
 
```

Comment:
Figure 7 shows the TTPs for the optical tunnel but I am not sure it would help here. I think the reader should be familiar with RFC8795 otherwise we risk to create another TE topology tutorial

Reply:
Rephrased

- [ ] @italobusi Update Figure 7 adding TTP identifiers (e.g., 11 on NE11) and the TTP symbol to the legend

Reply:
Added a new ad-hoc figure